app/src/main/java/com/cornellappdev/android/eatery/Model/MealMenuModel.java

```diff
@@ -0,0 +1,59 @@
+package com.cornellappdev.android.eatery.Model;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Model class to get menu for each meal
+ */
+
+public class MealMenuModel implements Serializable {
+    private Map<String, List<String>> menu = new HashMap<>();
+
+    public void addItem(String category, String item) {
+        if (menu.get(category) == null) {
+            menu.put(category, new ArrayList<String>());
+        }
+        menu.get(category).add(item);
+    }
+
+    public List<String> getItems(String category) {
+        return new ArrayList<>(menu.get(category));
```

Comment:
Can we add in handling for if an in valid category is given so we do not have null pointer exceptions in the ui side